vault/core.go

```diff
@@ -386,6 +389,9 @@ type Core struct {
 	// disabled
 	physicalCache physical.ToggleablePurgemonster
 
+	// logRequestsInfo indicates at which level requests should be logged
+	logRequestsInfo uberAtomic.String
```

Comment:
Why a string, making us have to call LevelFromString all the time?  Why not store the int result instead?